### PR TITLE
Update ticdc-architecture.md

### PR DESCRIPTION
**Pull request by IANTHEREAL**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [ ] master (the latest development version)
- [ ] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: ticdc/ticdc-architecture.md [hunk 1]
```diff
@@ -109,21 +109,21 @@ dispatchers = [
     对于 TiCDC 节点来说，TiKV 节点发送过来的 ResolvedTS 信息是一种特殊的事件，它只包含一个格式为 `<resolvedTS:  时间戳>` 的特殊事件。通常情况下，ResolvedTS 满足以下约束：
 
     ```
-    table resolved TS >= local resolved TS >= global Resolved TS
+    table resolved TS >= global Resolved TS
```
Comment (shichun-0415) [inline ticdc/ticdc-architecture.md:112]: Please unify:
table resolved TS vs table ResolvedTS?
global Resolved TS vs global ResolvedTS

File: ticdc/ticdc-architecture.md [hunk 2]
```diff
@@ -109,21 +109,21 @@ dispatchers = [
...
     ```
 
 #### CheckpointTS
 
 这个时间戳只在 TiCDC 中存在，它表示 TiCDC 已经同步给下游的数据的最低水位线，即 TiCDC 认为在这个时间戳之前的数据已经被同步到下游系统了。由于 TiCDC 同步数据的单位是表，所以 table checkpointTS 表示表级别的同步数据的水位线，Processor checkpointTS 表示各个 Processor 中最低的 checkpointTS；Global checkpointTS 表示各个 Processor checkpointTS 中最低的 checkpointTS。通常情况下，Checkpoint TS 满足以下约束：
 
 ```
-table checkpoint TS >= local checkpoint TS >= global checkpoint TS
+table checkpoint TS >= global checkpoint TS
```
Comment (shichun-0415) [inline ticdc/ticdc-architecture.md:120]: ditto

File: ticdc/ticdc-architecture.md [hunk 3]
```diff
@@ -109,21 +109,21 @@ dispatchers = [
...
 ```
 
-如果将 ResolvedTS 和 checkpointTS 结合来看，完整的关系可以表达为：
+因为 TiCDC 只会复制小于 global Resolved TS 的数据到下游，所以存在下面的约束：
 
 ```
-table resolved TS >= local resolved TS >= global Resolved TS >= table checkpoint TS >= local checkpoint TS >= global checkpoint TS
+table resolved TS >= global Resolved TS >= table checkpoint TS >= global checkpoint TS
```
Comment (shichun-0415) [inline ticdc/ticdc-architecture.md:126]: ditto

File: ticdc/ticdc-architecture.md [hunk 4]
```diff
@@ -137,7 +137,7 @@ table resolved TS >= local resolved TS >= global Resolved TS >= table checkpoint
 - 对于 DDL 语句，这个时间戳会用来确保在这个 DDL 语句之前的改变都被应用到下游，之后执行对应的 DDL 语句，在 DDL 语句同步完成之后再同步其他的数据改变。由于 DDL 语句的处理是 owner 角色的 Capture 负责的，DDL 语句对应的 Barrier TS 只会由 owner 节点的 Processor 线程产生。
 - sync point Barrier TS 也是一个时间戳，当你启用 TiCDC 的 Syncpoint 特性后，TiCDC 会根据你指定的间隔产生一个 Barrier TS，当所有的表都同步到了这个 Barrier TS 之后，记录一下对应的时间点，之后继续向下同步数据。
 
-TiCDC 是通过对 global checkpoint TS 和 barrier TS 进行比较来确定数据是否已经同步到 barrier TS 的。如果 global checkpoint TS = barrier TS，则说明所有表都至少推进到 barrier TS。如果等式不成立，说明有表还没推进到该 barrier TS，在这种情况下也不会更新 barrier TS，因此不会有表的 sink 节点对应 resolved TS 会超过 barrier TS，即不会有表的 check point TS 超过 barrier TS。
+TiCDC 产生 barrier TS 后, 会控制只复制小于 barrier TS 的数据下游。然后通过比较 global checkpoint TS 和 barrier TS 定小于 barrier TS 是否已经被同步完成。如果 global checkpoint TS = barrier TS，则说明所有小于 barrier TS 的数据都已经被同步完成，则可以开始继续同步小于 barrier TS 的数据。如果等式不成立，说明还有小于 barrier TS 的数据没有被同步到下游，TiCDC 则需要继续等待所有小于 barrier TS 数据的同步完成。
```
Comment (shichun-0415) [inline ticdc/ticdc-architecture.md:140]:
```suggestion
TiCDC 生成 barrier TS 后, 会只复制小于 barrier TS 的数据到下游。然后比较 global checkpoint TS 和 barrier TS 的大小，确定小于 barrier TS 的数据是否已经被同步完成。如果 global checkpoint TS = barrier TS，则说明所有小于 barrier TS 的数据都已经被同步完成，则可以开始继续同步小于 barrier TS 的数据。如果等式不成立，说明还有小于 barrier TS 的数据没有被同步到下游，TiCDC 则需要继续等待所有小于 barrier TS 数据的同步完成。
```

File: ticdc/ticdc-architecture.md (hunk 1 above)
Comment (IANTHEREAL) [inline ticdc/ticdc-architecture.md:112]:
```suggestion
    table resolved TS >= global resolved TS
```

File: ticdc/ticdc-architecture.md (hunk 3 above)
Comment (IANTHEREAL) [inline ticdc/ticdc-architecture.md:126]:
```suggestion
table resolved TS >= global resolved TS >= table checkpoint TS >= global checkpoint TS
```

File: ticdc/ticdc-architecture.md [hunk 5]
```diff
@@ -135,9 +135,9 @@ table resolved TS >= local resolved TS >= global Resolved TS >= table checkpoint
 当系统发生 DDL 语句或者使用了 TiCDC 的 Syncpoint 时会产生的一个时间戳。
 
 - 对于 DDL 语句，这个时间戳会用来确保在这个 DDL 语句之前的改变都被应用到下游，之后执行对应的 DDL 语句，在 DDL 语句同步完成之后再同步其他的数据改变。由于 DDL 语句的处理是 owner 角色的 Capture 负责的，DDL 语句对应的 Barrier TS 只会由 owner 节点的 Processor 线程产生。
-- sync point Barrier TS 也是一个时间戳，当你启用 TiCDC 的 Syncpoint 特性后，TiCDC 会根据你指定的间隔产生一个 Barrier TS，当所有的表都同步到了这个 Barrier TS 之后，记录一下对应的时间点，之后继续向下同步数据。
+- sync point 的 Barrier TS 也是一个时间戳，当你启用 TiCDC 的 Syncpoint 特性后，TiCDC 会根据你指定的间隔产生一个 Barrier TS，当所有的表都同步到了这个 Barrier TS 之后，记录一下对应的时间点，之后继续向下同步数据。
 
-TiCDC 是通过对 global checkpoint TS 和 barrier TS 进行比较来确定数据是否已经同步到 barrier TS 的。如果 global checkpoint TS = barrier TS，则说明所有表都至少推进到 barrier TS。如果等式不成立，说明有表还没推进到该 barrier TS，在这种情况下也不会更新 barrier TS，因此不会有表的 sink 节点对应 resolved TS 会超过 barrier TS，即不会有表的 check point TS 超过 barrier TS。
+一个 barrier TS 被生成后后, TiCDC 会只先复制小于 barrier TS 的数据到下游。然后比较 global CheckpointTS 和 Barrier TS 的大小，确定小于 Barrier TS 的数据是否已经被同步完成。如果 global CheckpointTS = Barrier TS，则可以开始同步大于 Barrier TS 的数据，否则需要继续等待所有小于 Barrier TS 数据的同步完成。
```
Comment (IANTHEREAL) [inline ticdc/ticdc-architecture.md:140]:
```suggestion
一个 Barrier TS 被生成后后, TiCDC 会只先复制小于 Barrier TS 的数据到下游。然后比较 global CheckpointTS 和 Barrier TS 的大小，确定小于 Barrier TS 的数据是否已经被同步完成。如果 global CheckpointTS = Barrier TS，则可以开始同步大于 Barrier TS 的数据，否则需要继续等待所有小于 Barrier TS 数据的同步完成。
```

File: ticdc/ticdc-architecture.md [hunk 6]
```diff
@@ -135,9 +135,9 @@ table resolved TS >= local resolved TS >= global Resolved TS >= table checkpoint
 当系统发生 DDL 语句或者使用了 TiCDC 的 Syncpoint 时会产生的一个时间戳。
 
 - 对于 DDL 语句，这个时间戳会用来确保在这个 DDL 语句之前的改变都被应用到下游，之后执行对应的 DDL 语句，在 DDL 语句同步完成之后再同步其他的数据改变。由于 DDL 语句的处理是 owner 角色的 Capture 负责的，DDL 语句对应的 Barrier TS 只会由 owner 节点的 Processor 线程产生。
-- sync point Barrier TS 也是一个时间戳，当你启用 TiCDC 的 Syncpoint 特性后，TiCDC 会根据你指定的间隔产生一个 Barrier TS，当所有的表都同步到了这个 Barrier TS 之后，记录一下对应的时间点，之后继续向下同步数据。
+- sync point 的 Barrier TS 也是一个时间戳，当你启用 TiCDC 的 Syncpoint 特性后，TiCDC 会根据你指定的间隔产生一个 Barrier TS，当所有的表都同步到了这个 Barrier TS 之后，记录一下对应的时间点，之后继续向下同步数据。
 
-TiCDC 是通过对 global checkpoint TS 和 barrier TS 进行比较来确定数据是否已经同步到 barrier TS 的。如果 global checkpoint TS = barrier TS，则说明所有表都至少推进到 barrier TS。如果等式不成立，说明有表还没推进到该 barrier TS，在这种情况下也不会更新 barrier TS，因此不会有表的 sink 节点对应 resolved TS 会超过 barrier TS，即不会有表的 check point TS 超过 barrier TS。
+一个 Barrier TS 被生成后后, TiCDC 会只先复制小于 Barrier TS 的数据到下游。然后比较 global CheckpointTS 和 Barrier TS 的大小，确定小于 Barrier TS 的数据是否已经被同步完成。如果 global CheckpointTS = Barrier TS，则可以开始同步大于 Barrier TS 的数据，否则需要继续等待所有小于 Barrier TS 数据的同步完成。
```
Comment (nongfushanquan) [inline ticdc/ticdc-architecture.md:140]: 一个 Barrier TS 被生成后后, TiCDC 会只先复制小于 Barrier TS 的数据到下游。然后比较 global CheckpointTS 和 Barrier TS 的大小，确定小于 Barrier TS 的数据是否已经被同步完成。如果 global CheckpointTS = Barrier TS，执行完对应的操作（如DDL）后，更新 Barrier TS  为下一个新值，否则需要继续等待所有小于 Barrier TS 数据的同步完成。

Comment (IANTHEREAL) [inline ticdc/ticdc-architecture.md:140]:
```suggestion
一个 Barrier TS 被生成后后, TiCDC 会只先复制小于 Barrier TS 的数据到下游。然后比较 global CheckpointTS 和 Barrier TS 的大小，确定小于 Barrier TS 的数据是否已经被同步完成。如果 global CheckpointTS = Barrier TS，执行完对应的操作（如 DDL 或者更新 sync point）后，产生下一个 Barrier TS，继续往前同步；否则需要继续等待所有小于 Barrier TS 数据的同步完成。
```

File: ticdc/ticdc-architecture.md [hunk 7]
```diff
@@ -135,9 +135,9 @@ table resolved TS >= local resolved TS >= global Resolved TS >= table checkpoint
 当系统发生 DDL 语句或者使用了 TiCDC 的 Syncpoint 时会产生的一个时间戳。
 
 - 对于 DDL 语句，这个时间戳会用来确保在这个 DDL 语句之前的改变都被应用到下游，之后执行对应的 DDL 语句，在 DDL 语句同步完成之后再同步其他的数据改变。由于 DDL 语句的处理是 owner 角色的 Capture 负责的，DDL 语句对应的 Barrier TS 只会由 owner 节点的 Processor 线程产生。
-- sync point Barrier TS 也是一个时间戳，当你启用 TiCDC 的 Syncpoint 特性后，TiCDC 会根据你指定的间隔产生一个 Barrier TS，当所有的表都同步到了这个 Barrier TS 之后，记录一下对应的时间点，之后继续向下同步数据。
+- sync point 的 Barrier TS 也是一个时间戳，当你启用 TiCDC 的 Syncpoint 特性后，TiCDC 会根据你指定的间隔产生一个 Barrier TS，当所有的表都同步到了这个 Barrier TS 之后，记录一下对应的时间点，之后继续向下同步数据。
```
Comment (IANTHEREAL) [inline ticdc/ticdc-architecture.md:138]:
```suggestion
- sync point 的 Barrier TS 也是一个时间戳，当你启用 TiCDC 的 Syncpoint 特性后，TiCDC 会根据你指定的间隔产生一个 Barrier TS，当所有的表都同步到了这个 Barrier TS 之后，记录 CheckpointTS 到下游，之后继续向下同步数据。
```

**Review by shichun-0415 — APPROVED** [review]

**Pull request merged**